src/datamodel/client-state.ts

```diff
@@ -43,7 +43,8 @@ export const userInfoSchema = z.object({
 });
 
 export const clientStateSchema = z.object({
-  id: z.string(),
+  clientID: z.string(),
+  id: z.literal(""),
```

Comment:
This one is a bit strange. 

```js
id: z.string(),
```

works too but `''` is more accurate

Reply:
Why does it need to be included? Can't the API support not including an ID if not necessary (and defaulting it to empty string)?

Reply:
OK. With the new PR (https://github.com/rocicorp/rails/pull/25) there is no need for this dummy property.